translations/ko.po

```diff
@@ -848,47 +848,47 @@ msgstr "패스워드"
 #: src\commands\search.ts:56
 msgctxt "command-option"
 msgid "konami-id"
-msgstr ""
+msgstr "코나미 ID"
```

Comment:
The other test failure is from the space here.

Reply:
ok. i fix this.